src/components/atoms/Heading/Heading.scss

```diff
@@ -0,0 +1,17 @@
+@font-face {
```

Comment:
I would suggest importing fonts once into App.scss and not to import them every scss file separately

Reply:
Changed